actions/common.go

```diff
@@ -26,6 +26,7 @@ var (
 		"router":           {"Router"},
 		"slugbuilder":      {"SlugBuilder"},
 		"slugrunner":       {"SlugRunner"},
+    "stdout-metrics":   {"StdoutMetrics"}
```

Comment:
also, can this be spaced the same as the others? if GH is showing it weird, ignore me